### PR TITLE
Document New Controllers and Gazebo Screen on the Setup Assistant 

### PR DESCRIPTION
**Pull request by MohmadAyman**
This updates the `moveit_setup_assistant` tutorial, showing the user that they should be able to spawn their robot into gazebo, in the case of this tutorial, `franka` robot.

This should be merged after [PR#956](https://github.com/ros-planning/moveit/pull/956) is merged.

### CODE REVIEW

**Comment by mlautman** [comment]
@MohmadAyman Please add a description for this PR

File: doc/setup_assistant/setup_assistant_tutorial.rst [hunk 1]
```diff
@@ -275,6 +275,26 @@ files that you will need to start using MoveIt!
...
   files you need for MoveIt!
 
+
+Gazebo Simulation
+------------------------------------
+
+You can use the generated robot urdf file in the simulation folder to
+spawn the robot in Gazebo.
+
+* Use rosrun to start gazebo empty world: ::
```
Comment (davetcoleman) [inline doc/setup_assistant/setup_assistant_tutorial.rst]: Make ``rosrun`` have single quotes (`) around it so it looks like a code command

File: doc/setup_assistant/setup_assistant_tutorial.rst [hunk 2]
```diff
@@ -275,6 +275,26 @@ files that you will need to start using MoveIt!
 * Congratulations!! - You are now done generating the configuration
   files you need for MoveIt!
 
+
```
Comment (davetcoleman) [inline doc/setup_assistant/setup_assistant_tutorial.rst]: Add documentation above this for how to use the new controller screen

File: doc/setup_assistant/setup_assistant_tutorial.rst [hunk 3]
```diff
@@ -275,6 +275,26 @@ files that you will need to start using MoveIt!
...
+------------------------------------
+
+You can use the generated robot urdf file in the simulation folder to
+spawn the robot in Gazebo.
+
+* Use rosrun to start gazebo empty world: ::
+
+   roslaunch gazebo_ros empty_world.launch paused:=true use_sim_time:=false gui:=true throttled:=false recording:=false debug:=true
+
+* Use rosrun to spawn the robot: ::
```
Comment (davetcoleman) [inline doc/setup_assistant/setup_assistant_tutorial.rst]: shouldn't this be a roslaunch file instead?

**Comment by davetcoleman** [comment]
I think the title could be better, e.g. "Document New Controllers and Gazebo Screen on MSA"

File: doc/setup_assistant/setup_assistant_tutorial.rst (hunk 1 above)
Comment (davetcoleman) [reply doc/setup_assistant/setup_assistant_tutorial.rst]: ping: these requested changes

File: doc/setup_assistant/setup_assistant_tutorial.rst (hunk 3 above)
Comment (MohmadAyman) [reply doc/setup_assistant/setup_assistant_tutorial.rst]: I think it is a `rosrun` at least according to the [`roslaunch` tutorial](http://gazebosim.org/tutorials?tut=ros_roslaunch) .

File: doc/setup_assistant/setup_assistant_tutorial.rst (hunk 1 above)
Comment (MohmadAyman) [reply doc/setup_assistant/setup_assistant_tutorial.rst]: Done

**Comment by MohmadAyman** [comment]
Do we wait until all the new screens have been merged and update the docs once? to also update the screenshots with the new tabs in the nav bar.

**Comment by davetcoleman** [comment]
sgtm

**Comment by davetcoleman** [comment]
Because everything has been merged in, its time for this to go live

**Pull request merged**